README.md

```diff
@@ -17,6 +17,9 @@ Learn more at  [Zenko.io](http://www.zenko.io/).
 
 ## Contributing
 
+You can find the roadmap, features in development and bugs for the release 1.2 on [Zenko Jira Project](https://scality.atlassian.net/projects/ZENKOIO/issues/ZENKOIO-19?filter=allopenissues) as well as for the future releases.
```

Comment:
The only thing I'm not sure about is having a specific release. It would have to be updated for every new one, which is okay as long as we stay on top of it. The other option could be to just say something like `the next release`.

Reply:
good point :)